deploy/olm-catalog/kubevirt-hyperconverged/1.4.0/kubevirt-hyperconverged-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     categories: OpenShift Optional
     certified: "false"
     containerImage: quay.io/kubevirt/hyperconverged-cluster-operator@sha256:a1b248457d1d6296390e7aa8b92e3db2a7786f32fe483ad9f44ca40ad1d69280
-    createdAt: "2021-01-20 12:20:36"
+    createdAt: "2021-01-21 09:41:24"
```

Comment:
if this is the only change in this file, please cancle this change to avoid conflicts.